tests/Test.ADAL.NET.Unit/OboFlowTests.cs

```diff
@@ -581,7 +587,8 @@ public void ResetInstanceDiscovery()
                 PostData = new Dictionary<string, string>()
```

Comment:
PostData seems to be cloned for multiple test cases. Consider adding to a helper or static in this test for re-use

Reply:
PostData represents data send in Auth request, It might be different for different test cases, also it is one of the main things to test (by looking at test case It should very clear what parameter are send in auth request) so moving  initialization out of the test might decrease readability